helm-charts/basehub/values.jsonnet

```diff
@@ -14,15 +14,20 @@ local emitDaskHubCompatibleConfig(basehubConfig) =
 
 local jupyterhubHomeNFSResources = {
   quotaEnforcer: {
-    path: '/export/%s' % hub_name,
+    config: {
+      QuotaManager: {
+        paths: ['/export/%s' % hub_name],
+        hard_quota: 0,
```

Comment:
This means no quota. Without this, all hubs got a "10GB" quota by default from the `jupyterhub-home-nfs` chart.